### PR TITLE
Updates to the Wikidata engineering team

### PR DESCRIPTION
**Pull request by Ladsgroup**
- Adding new hires
 - Adding Wikimedia hackathon in Prague

### CODE REVIEW

**Comment by Ladsgroup** [comment]
Alaa is not in https://wikimedia.de/de/menschen/mitarbeitende :/ We should fix that.

File: app/templates/pages/department.html [hunk 1]
```diff
@@ -123,6 +124,16 @@ <h4>Thiemo Kreuz<br>
...
                             <h3>Wikidata Team</h3>
                         </div>
+                        <div class="col-lg-4 col-md-6 text-center">
+                            <div class="thumbnail">
+                                <img class="img-responsive" src="https://wikimedia.de/img/staff/default.jpg" alt="">
+                                <div class="caption">
+                                    <h4>Michael Große<br>
+                                        <small>Software Developer</small>
+                                    </h4>
+                                </div>
```
Comment (micgro42) [inline app/templates/pages/department.html]: Feel free to add my wikimedia email and github here as well:

```suggestion
                                    <ul class="list-inline">
                                        <li><a href="mailto:michael.grosse@wikimedia.de"><i class="fa fa-2x fa-envelope-square"></i></a></li>
                                        <li><a href="https://github.com/micgro42"><i class="fa fa-2x fa-github-square"></i></a></li>
                                    </ul>
                                </div>
```

Comment (micgro42) [reply app/templates/pages/department.html]: (we can do this in another pr as well, when we are adding the pictures)

Comment (Ladsgroup) [reply app/templates/pages/department.html]: I'm adding way more people :D We can do that later too.

**Pull request merged**